Main.java

```diff
@@ -9,7 +10,20 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+    Scanner input = new Scanner(System.in);
+    // Asking for quantities 
+    System.out.println("How many chromebook chargers?");
+     int chargers = input.nextInt();
+      System.out.println("How many motherboards?");
+      int motherboards = input.nextInt();
+      System.out.println("How many mice?");
+      int mice = input.nextInt();
+      // calculating basic subtotal
+double subtotal = chargers*34.99+motherboards*127.50+mice*18.00;
+// stating subtotal
+System.out.println("Subtotal= $" + subtotal);
+//stating and calculating taxes and total with taxes
+System.out.println("Subtotal= $" + subtotal*0.13);
+System.out.println("Subtotal= $" + subtotal*1.13);

```

Comment:
The math is correct here but they are not all subtotals.... Also, we normally don't like to have math in print statements. Instead, store the math in a variable and use that variable in the output.